src/components/Header/HeaderPopover.tsx

```diff
@@ -57,7 +71,7 @@ function HeaderPopover({
           {({ height }) => (
             <AnimatedDiv
               ref={popoverFocusElement}
-              tabIndex="0"
+              tabIndex={0}
```

Comment:
we changed the type here, can we just double check if this is working properly or not?

Reply:
Nope, already did that. We have more types problems there.